conjure-java-core/src/test/java/com/palantir/conjure/java/UndertowServiceEteTest.java

```diff
@@ -389,75 +374,57 @@ public void testOptionsOnGetIncludesHead() throws IOException {
         assertThat(con.getHeaderField(HttpHeaders.ALLOW)).isEqualTo("OPTIONS, GET, HEAD");
     }
 
-    @Test
-    public void testUnknownContentType() {
-        assertThatThrownBy(() -> {
```

Comment:
We should retain this test, but use `HttpURLConnection` instead of retrofit (verifying the response status code, and SerializableError response body)